changelog/unreleased/enhancement-search-content-extraction-cleanup.md

```diff
@@ -0,0 +1,13 @@
+Enhancement: Tika content extraction cleanup for search
+
+So far it has not been possible to determine whether
+the content for the search should be cleaned of stop words or not.
+
+This can now be set with the newly introduced settings option `SEARCH_EXTRACTOR_TIKA_CLEAN_STOP_WORDS=false`
+which is enabled by default.
```

Comment:
```suggestion
The behaviour can now be set with the newly introduced settings option `SEARCH_EXTRACTOR_TIKA_CLEAN_STOP_WORDS=false`
which is enabled by default.
```
The senctence is not clear with respect to enabled or not.
Is `false` enabled? because the envvar default is `true` and that makes headaches.